daphne/ws_protocol.py

```diff
@@ -54,6 +56,15 @@ def onConnect(self, request):
             else:
                 self.client_addr = None
                 self.server_addr = None
+
+            if self.factory.proxy_forwarded_address_header:
```

Comment:
It looks like this should have been `if self.main_factory.proxy_forward_address_header:`?

I'm getting lots of _WebSocketFactory object has no attribute proxy_forwarded_address_header_ errors with this version of Daphne.